src/app/services/main-form-manager/config-map-export-helper.ts

```diff
@@ -484,6 +521,52 @@ export class ConfigMapExportHelper {
     }
   }
 
+  /**
+   *  Methode used to construct the key props
+   */
+  public static getCircleHeatMapSortKey(fgValues: any): PaintValue {
+    switch (fgValues.propertySource) {
+      case PROPERTY_SELECTOR_SOURCE.fix_color:
+        return 1;
+      case PROPERTY_SELECTOR_SOURCE.interpolated:
+        const interpolatedValues = fgValues.propertyInterpolatedFg as InterpolatedProperty;
+        const minValue = interpolatedValues.propertyInterpolatedValuesCtrl[0].proportion;
+        const maxValue = interpolatedValues
+          .propertyInterpolatedValuesCtrl[interpolatedValues.propertyInterpolatedValuesCtrl.length - 1]
+          .proportion;
+        return [
+          'interpolate',
+          [
+            'linear'
+          ],
+          [
+            'get',
```

Comment:
we should sort the values of the chosen metric for the colorFg (not necessarily the count)